packages/driver/cypress/e2e/commands/traversals.cy.js

```diff
@@ -160,58 +160,25 @@ describe('src/cy/commands/traversals', () => {
           cy.get('#list')[name](arg)
         })
 
-        it('snapshots after finding element', () => {
-          cy.get('#list')[name](arg).then(function () {
-            const { lastLog } = this
-
-            expect(lastLog.get('snapshots').length).to.eq(1)
-            expect(lastLog.get('snapshots')[0]).to.be.an('object')
-          })
-        })
-
-        it('has the $el', () => {
+        it('snapshots, has the element, and logs with console props', () => {
```

Comment:
No functional changes here, but combining the assertions into one test ends up saving about 60s per run on my machine (since this is executed in a loop), and likely more than that on a CI box.

No need to revisit dom.html hundreds of times. :woman_shrugging: